### DISCUSSION TITLE
"npx create-abell-app ." not working.

### DISCUSSION THREAD

**Discussion by iamsriranga** [abelljs/abell#103] [Q&A] [answered]
"npx create-abell-app ." not working, is this intensional or should we need to add this feature.
I don't have any other project with the same name. 

@saurabhdaware 

![Screenshot (286)](https://user-images.githubusercontent.com/43727167/101672468-e8706300-3a7b-11eb-8c82-b3676fd5280f.png)

**Comment by saurabhdaware** [comment]
So the format of the command is `npx create-abell-app <project-name>` In your case, you're giving dot as a project name.

In file system dot refers to the current directory and the current directory exists

Right way to run the command would be `npx create-abell-app my-cool-app`

**Reply by saurabhdaware** [reply]
We can also just run `npx create-abell-app` and it will ask you to enter the project name

**Reply by iamsriranga** [reply]
> 
> 
> So the format of the command is `npx create-abell-app <project-name>` In your case, you're giving dot as a project name.
> 
> In file system dot refers to the current directory and the current directory exists
> 
> Right way to run the command would be `npx create-abell-app my-cool-app`

But I wanted to run the create-abell-app in the current empty dir, like that of "c:/user/desktop/react-demo/ npx create-react-app .", I think you get it.

**Reply by iamsriranga** [reply]
> 
> 
> > So the format of the command is `npx create-abell-app <project-name>` In your case, you're giving dot as a project name.
> > In file system dot refers to the current directory and the current directory exists
> > Right way to run the command would be `npx create-abell-app my-cool-app`
> 
> But I wanted to run the create-abell-app in the current empty dir, like that of "c:/user/desktop/react-demo/ npx create-react-app .", I think you get it.

How about adding this feature? @saurabhdaware

**Reply by saurabhdaware** [reply]
Oh got it! How does `create-react-app` handles this case? 

So currently we just check if the directory exists. Further I think we can check if the directory is empty or not. and if it is empty, we can continue the process.

**Reply by iamsriranga** [reply]
> 
> 
> Oh got it! How does `create-react-app` handles this case?
> 
> So currently we just check if the directory exists. Further I think we can check if the directory is empty or not. and if it is empty, we can continue the process.

Now you got it, Can I try this?

**Reply by saurabhdaware** [reply]
Yup makes sense to add this! The code is in the https://github.com/abelljs/create-abell-app if you're interested in contributing 🥳

**Reply by iamsriranga** [reply]
@saurabhdaware, In this case, I need to look for the "." argument right? and then check if any other file exists in currDir.

**Comment by saurabhdaware** [comment, marked as answer]
Since this is an issue, The further conversation is at https://github.com/abelljs/create-abell-app/issues/16
